src/hardware/vga_memory.cpp

```diff
@@ -260,15 +263,15 @@ class VGA_UnchainedEGA_Handler : public VGA_UnchainedRead_Handler {
 	void writeHandler(PhysPt start, uint8_t val) {
 		uint32_t data=ModeOperation(val);
 		/* Update video memory and the pixel buffer */
-		VGA_Latch pixels;
+		VgaLatch pixels;
 		pixels.d=((uint32_t*)vga.mem.linear)[start];
 		pixels.d&=vga.config.full_not_map_mask;
 		pixels.d|=(data & vga.config.full_map_mask);
 		((uint32_t*)vga.mem.linear)[start]=pixels.d;
 		uint8_t * write_pixels=&vga.fastmem[start<<3];
 
 		uint32_t colors0_3, colors4_7;
-		VGA_Latch temp;temp.d=(pixels.d>>4) & 0x0f0f0f0f;
+		VgaLatch temp;temp.d=(pixels.d>>4) & 0x0f0f0f0f;
```

Comment:
Style: missing spaces around `=` and `>>`, assignment should go to the next line.

Reply:
Just a rename.